i18n/ads-language-pack-de/package.json

```diff
@@ -2,7 +2,7 @@
 	"name": "ads-language-pack-de",
 	"displayName": "German Language Pack for Azure Data Studio",
 	"description": "Language pack extension for German",
-	"version": "1.38.0",
+	"version": "1.39.0",
```

Comment:
I will change it back to 1.38, this is a mistake

Reply:
So I feel like we should work on the process here, because currently doing these updates from main risks the strings being different from what is actually in the 1.38 branch. 

Wouldn't we just want to do these updates directly to the 1.38 branch? Given that we only release these when a release is happening it seems fine to have that not be fully updated (although doing so wouldn't be bad in case anyone wants to test the latest langpacks themselves)